deis/logger
# feat(model/message_handler): Add time to log message output

Testing Instructions:

* clone this PR
* `make build push upgrade`
* checkout the corresponding fluentd PR - https://github.com/deis/fluentd/pull/61
* `make build push upgrade` in the fluentd directory
* Deploy an application that you can use to generate log entries with. 
* Use `deis logs` to see new message format.

The log format should look similar to this - 

```
2016-10-18T20:24:06+00:00 app[foo.web.v4]: 2016/10/18 20:24:05 10.240.0.7 GET / - 6334824724549167320
2016-10-18T20:24:07+00:00 app[foo.web.v4]: 2016/10/18 20:24:06 10.240.0.7 GET / - 605394647632969758
2016-10-18T20:24:07+00:00 app[foo.web.v4]: 2016/10/18 20:24:07 10.240.0.7 GET / - 1443635317331776148
2016-10-18T20:24:08+00:00 app[foo.web.v4]: 2016/10/18 20:24:07 10.164.2.1 GET / - 894385949183117216
2016-10-18T20:24:08+00:00 app[foo.web.v4]: 2016/10/18 20:24:07 10.240.0.5 GET / - 2775422040480279449
2016-10-18T20:29:03+00:00 deis[controller.INFO]: buildpack type detected. Defaulting to $PORT 5000
2016-10-18T20:29:03+00:00 deis[controller.INFO]: buildpack type detected. Defaulting to $PORT 5000
2016-10-18T20:29:03+00:00 deis[controller.INFO]: scaling Deployment foo-web from 1 to 4 replicas
2016-10-18T20:29:04+00:00 deis[controller.INFO]: adding 30s on to the original 120s timeout to account for the initial delay specified in the liveness / readiness probe
2016-10-18T20:29:04+00:00 deis[controller.INFO]: This deployments overall timeout is 300s - batch timout is 150s and there are 2 batches to deploy with a total of 4 pods
2016-10-18T20:29:14+00:00 deis[controller.INFO]: waited 10s and 4 pods are in service
2016-10-18T20:29:24+00:00 deis[controller.INFO]: waited 20s and 4 pods are in service
2016-10-18T20:29:34+00:00 deis[controller.INFO]: waited 30s and 4 pods are in service
2016-10-18T20:29:38+00:00 deis[controller.INFO]: admin scaled pods web=4
```
